docs/source/ContributionGuide.rst

```diff
@@ -228,6 +229,18 @@ Use it::
    You can not use the merge button on github to do signed merges as github
    does not have your private key.
 
+Licensing
+---------
+
+Except where otherwise indicated in a given source file, all original
+contributions to Avocado are licensed under the GNU General Public
+License version 2 `(GPLv2) <https://www.gnu.org/licenses/gpl-2.0.html>`_.
```

Comment:
I'm following the *official* license name (GPLv2).  I can't find "GPLv2+" in any part of official license text of official (gnu.org) references to it.

About the "any later version", that is indeed given as example in the license text.  So, I suggest we write the first block as:

"Except where otherwise indicated in a given source file, all original contributions to Avocado are licensed under the GNU General Public License version 2 (GPLv2) or any later version".

Comments?